src/shared/ui/Input/Input.tsx

```diff
@@ -31,8 +31,8 @@ export function Input({ HTMLType, iconType, placeholder, extraClass, defaultValu
 
   return (
     <>
-      <div className={`w-80 bg-custom-gray rounded-[10px] ${extraClass}`}>
-        <div className='flex items-center w-full h-50 overflow-hidden p-3.5'>
+      <div className={`w-80 bg-custom-gray rounded-[10px] border ${extraClass} ${error ? 'border-input-error' : 'border-transparent'}`}>
+        <div className='flex items-center w-full h-48px overflow-hidden p-3.5'>
```

Comment:
I think tailwind const should be called 48, not 48px

Reply:
I've decided to use '48px' because '48' already exists and I don't want ro overrride it in case somebody uses it